cndt-2019/README.md

```diff
@@ -0,0 +1,18 @@
+# CNDT 2019: New Kubernetes Contributor Workshop
+
+<img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png" width="200" style="float:left">
+<img src="https://cloudnativedays.jp/cndt2019/assets/img/preview/about-sm-1.png" width="200" style="float:left">
+
+Welcome to Kubernetes Upstream Training!!
+July 22th 2019 This is the location of our CNDT 2019 Kubernetes Upstream Training activities.
+
+## Docs
+
+Get started with Kubernbetes documentation
```

Comment:
add link to https://github.com/kubernetes/community/tree/master/contributors/guide

Reply:
OK. I'll fix.